### PR TITLE
[Doc] Add Python docstring for some operators

### PR DESCRIPTION
**Pull request by beginlner**
Related issue = #2579

### CODE REVIEW

File: python/taichi/lang/ops.py [hunk 1]
```diff
@@ -185,36 +185,92 @@ def _ternary_operation(taichi_op, python_op, a, b, c):
...
 @unary
 def neg(a):
+    """Negate function.
+
+    Args:
+        a: a.
+
+    Returns:
+        -a.
+    """
```
Comment (ljcc0930) [inline python/taichi/lang/ops.py:195]:
```suggestion
    """Negate function.

    Args:
        a: A number.

    Returns:
        The negative value of number a.
    """
```
```suggestion
        
```
Explaining with some words?

**Comment by ljcc0930** [comment]
Explaining the functions with some words?

**Comment by beginlner** [comment]
> Explaining the functions with some words?
> 
> e.g.:
> 
> ```
>     """Negate function.
>     Args:
>         a: A number.
>     Returns:
>         The negative value of number a.
>     """
> ```

ok

**Review by Leonz5288 — CHANGES_REQUESTED** [review]
I believe ti.field is better than ti.Expr which is pretty vague about the type being passed in. Also, ti.field is in parallel relationship with ti.Matrix. @ljcc0930

**Comment by strongoier** [comment]
> I believe ti.field is better than ti.Expr which is pretty vague about the type being passed in. Also, ti.field is in parallel relationship with ti.Matrix. @ljcc0930

The type of `ti.field` is `ti.Expr` for now. I'm currently refactoring this so the confusion about these types may be resolved in the future.

**Review by ljcc0930 — APPROVED** [review]
LGTM!

File: python/taichi/lang/ops.py [hunk 2]
```diff
@@ -185,36 +185,92 @@ def _ternary_operation(taichi_op, python_op, a, b, c):
 
 @unary
 def neg(a):
+    """The negate function.
+
+    Args:
+        a (Union[ti.Expr, ti.Matrix]): A number or a maxtrix.
```
Comment (ljcc0930) [inline python/taichi/lang/ops.py:191]:
```suggestion
        a (Union[:class:`~taichi.lang.expr.Expr`, :class:`~taichi.lang.matrix.Matrix`]): A number or a matrix.
```

File: python/taichi/lang/ops.py [hunk 3]
```diff
@@ -185,36 +185,92 @@ def _ternary_operation(taichi_op, python_op, a, b, c):
...
+    """
     return _unary_operation(_ti_core.expr_neg, _bt_ops_mod.neg, a)
 
 
 @unary
 def sin(a):
+    """The sine function.
+
+    Args:
+        a (Union[ti.Expr, ti.Matrix]): A number or a maxtrix.
```
Comment (ljcc0930) [inline python/taichi/lang/ops.py:204]:
```suggestion
        a (Union[:class:`~taichi.lang.expr.Expr`, :class:`~taichi.lang.matrix.Matrix`]): A number or a matrix.
```

**Review by k-ye — APPROVED** [review]
Great!

**Pull request merged**